modules/nf-core/crumble/main.nf

```diff
@@ -0,0 +1,50 @@
+process CRUMBLE {
+    tag "$meta.id"
+    label 'process_medium'
+
+    conda "bioconda::crumble=0.9.1"
+    container "${ workflow.containerEngine == 'singularity' && !task.ext.singularity_pull_docker_container ?
+        'https://depot.galaxyproject.org/singularity/crumble:0.9.1--hb0d9459_0':
+        'quay.io/biocontainers/crumble:0.9.1--hb0d9459_0' }"
+
+    input:
+    tuple val(meta), path(input)
+    path keepbed
+    val bedout
+
+    output:
+    tuple val(meta), path("*.bam"),  emit: bam,     optional: true
+    tuple val(meta), path("*.cram"), emit: cram,    optional: true
+    tuple val(meta), path("*.sam"),  emit: sam,     optional: true
+    tuple val(meta), path("*.bed"),  emit: bed,     optional: true
+    path "versions.yml",             emit: versions
+
+    when:
+    task.ext.when == null || task.ext.when
+
+    script:
+    def args       = task.ext.args ?: ''
+    def prefix     = task.ext.prefix ?: "${meta.id}"
+    def extension  = args.contains("-O sam") ? "sam" :
+                    args.contains("-O bam") ? "bam" :
+                    args.contains("-O cram") ? "cram" :
+                    "sam"
+    def bedin      = keepbed ? "-R ${keepbed}" : ""
+    def bedout     = bedout ? "-b ${prefix}.out.bed" : ""
```

Comment:
I would suggest renaming this BED file to make the name more informative. The help page says
```
-b out.bed        Output suspicious regions to out.bed []
```
So maybe `${prefix}.suspicious_regions.bed` ?

Reply:
Done ✅